protocol/grpc/grpc_invoker.go

```diff
@@ -43,14 +46,23 @@ type GrpcInvoker struct {
 	quitOnce    sync.Once
 	clientGuard *sync.RWMutex
 	client      *Client
+	// timeout for service(interface) level.
+	timeout time.Duration
 }
 
 // NewGrpcInvoker returns a Grpc invoker instance
 func NewGrpcInvoker(url *common.URL, client *Client) *GrpcInvoker {
+	requestTimeout := config.GetConsumerConfig().RequestTimeout
+	requestTimeoutStr := url.GetParam(constant.TIMEOUT_KEY, config.GetConsumerConfig().Request_Timeout)
+	if t, err := time.ParseDuration(requestTimeoutStr); err == nil {
```

Comment:
```suggestion
	if t, err := time.ParseDuration(requestTimeoutStr); err != nil {
```
Is it a right logic?